test/shared/src/main/scala/zio/test/DefaultTestReporter.scala

```diff
@@ -144,9 +146,41 @@ object DefaultTestReporter {
   private def renderCause(cause: Cause[Any], offset: Int): String =
     cause match {
```

Comment:
You should be able to just use `untraced` here and get rid of the `Cause.Traced` in your pattern match so you display the mock exceptions the way you want regardless of whether tracing is enabled.